checker/jtreg/nullness/AssignmentPerformanceTest.java

```diff
@@ -1,6 +1,9 @@
-// @skip-test
-// Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
-
+/*
+ * @test
+ * @summary Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
+ *
+ * @compile/timeout=60 -J-Xmx256M -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AprintErrorStack -Alint AssignmentPerformanceTest.java
```

Comment:
To confirm my change doesn't make terrible memory usage, I've added `-Xmx256M` option for JVM.